test/pathforaSpec.js

```diff
@@ -2195,6 +2195,49 @@ describe('Widgets', function () {
   //  DISPLAY CONDITIONS
   // -------------------------
 
+  it('should show when all manualTrigger widgets are triggered', function () {
+    $(document.body).append('<div class=\'hover-elem\'>Test</div>');
+
+    $('.hover-elem').hover(function () {
+      pathfora.triggerWidgets();
+    });
```

Comment:
Involving the DOM in this test seems unnecessary to me. Why not simply call `pathfora.triggerWidgets()` below instead of triggering the mouse events?